### PR TITLE
Semaphore v2 migration / add pin update pipeline

### PR DESCRIPTION
**Pull request by Brian-McM**
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by bcreane — APPROVED** [review]
/lgtm

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

File: .semaphore/clear_cache.yml
Comment (bcreane) [inline .semaphore/clear_cache.yml:5]: 2 hours seems too long.

File: .semaphore/semaphore.yml
Comment (bcreane) [inline .semaphore/semaphore.yml:5]: same here - looks like typical job is under 15 minutes, maybe set to `minutes: 30` for e.g.?

**Review by bcreane — APPROVED** [review]
couple of nits, but otherwise /lgtm.

File: .semaphore/semaphore.yml
Comment (Brian-McM) [reply .semaphore/semaphore.yml:5]: Ya good point, I'll change it.

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
